homeassistant/components/notify/manifest.json

```
@@ -2,6 +2,7 @@
   "domain": "notify",
   "name": "Notifications",
   "codeowners": ["@home-assistant/core"],
+  "dependencies": ["http", "repairs"],
```

Repairs depends on http.
```suggestion
  "dependencies": ["repairs"],
```